src/gui/tray/activitydata.cpp

```diff
@@ -50,6 +50,22 @@ OCC::Activity Activity::fromActivityJson(const QJsonObject &json, const AccountP
 {
     const auto activityUser = json.value(QStringLiteral("user")).toString();
 
+    const auto stringToUrl = [&] (const QString link) {
```

Comment:
1. Likely, you don't need `[&]` since you just need to capture `account`
2. `//QUrl::fromUserInput` is a leftover and needs to be removed
3. `const QString link` would look better like this `const QString &link`